hedera-node/hedera-app-spi/build.gradle.kts

```diff
@@ -29,7 +29,5 @@ testModuleInfo {
     requires("org.assertj.core")
     requires("org.junit.jupiter.api")
     requires("org.junit.jupiter.params")
-    requires("org.mockito")
-    requires("org.mockito.junit.jupiter")
```

Comment:
These are not needed. Looks like this is wrong on `develop`. When leaving these in, check is failing on this PR.